<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>frmBarScale</class>
 <widget class="QWidget" name="frmBarScale">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>500</width>
    <height>300</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QHBoxLayout" name="horizontalLayout">
   <item>
    <widget class="BarScale" name="widget1">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
       <horstretch>3</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="maxValue">
      <double>800.000000000000000</double>
     </property>
     <property name="maxValueRight">
      <double>800.000000000000000</double>
     </property>
     <property name="longStep">
      <number>200</number>
     </property>
     <property name="shortStep">
      <number>20</number>
     </property>
     <property name="longStepRight">
      <number>200</number>
     </property>
     <property name="shortStepRight">
      <number>20</number>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QSlider" name="verticalSlider1">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
    </widget>
   </item>
   <item>
    <widget class="BarScale" name="widget2">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
       <horstretch>4</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="maxValue">
      <double>800.000000000000000</double>
     </property>
     <property name="longStep">
      <number>200</number>
     </property>
     <property name="shortStep">
      <number>20</number>
     </property>
     <property name="longStepRight">
      <number>20</number>
     </property>
     <property name="shortStepRight">
      <number>5</number>
     </property>
     <property name="scaleStyle">
      <enum>BarScale::ScaleStyle_Double</enum>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QSlider" name="verticalSlider2">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
    </widget>
   </item>
   <item>
    <widget class="BarScale" name="widget3">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
       <horstretch>3</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QSlider" name="verticalSlider3">
     <property name="maximum">
      <number>100</number>
     </property>
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>BarScale</class>
   <extends>QWidget</extends>
   <header>barscale.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections>
  <connection>
   <sender>verticalSlider3</sender>
   <signal>valueChanged(int)</signal>
   <receiver>widget3</receiver>
   <slot>setValue(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>482</x>
     <y>194</y>
    </hint>
    <hint type="destinationlabel">
     <x>449</x>
     <y>195</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>verticalSlider3</sender>
   <signal>valueChanged(int)</signal>
   <receiver>widget3</receiver>
   <slot>setValueRight(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>479</x>
     <y>241</y>
    </hint>
    <hint type="destinationlabel">
     <x>457</x>
     <y>238</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
